src/query/storages/fuse/src/operations/mutation/compact/compact_part.rs

```diff
@@ -13,23 +13,67 @@
 // limitations under the License.
 
 use std::any::Any;
+use std::collections::hash_map::DefaultHasher;
+use std::hash::Hash;
+use std::hash::Hasher;
 use std::sync::Arc;
 
 use common_catalog::plan::PartInfo;
 use common_catalog::plan::PartInfoPtr;
 use common_exception::ErrorCode;
 use common_exception::Result;
 use storages_common_table_meta::meta::BlockMeta;
+use storages_common_table_meta::meta::CompactSegmentInfo;
+use storages_common_table_meta::meta::Statistics;
 
 use crate::operations::common::BlockMetaIndex;
+use crate::operations::mutation::BlockIndex;
+use crate::operations::mutation::SegmentIndex;
+
+#[derive(serde::Serialize, serde::Deserialize, PartialEq, Clone)]
+pub struct CompactLazyPartInfo {
+    pub segment_indices: Vec<SegmentIndex>,
+    pub compact_segments: Vec<Arc<CompactSegmentInfo>>,
+}
```

Comment:
let me see, CompactLazyPartInfo is used to see if this execution is distributed, right?  sorry, I'm a little confused, what's the difference of CompactLazyPartInfo and CompactPartInfo?

Reply:
Just like `FuseLazyPartInfo` and `FusePartInfo`.